### ISSUE TITLE
The elements render only after pressing F5

### ISSUE THREAD

**Issue by benessto** [CherryProjects/react-facebook#30]
The like button only renders when I press F5. After changing the component to go to another page, it's the same without the button

Here is a video to show:
https://i.gyazo.com/6c5cd185fe1437142ce19a231e4db3aa.mp4

**Comment by seeden** [comment]
@benessto what is your browser? Can you paste your source code?

**Comment by seeden** [comment]
I released new updated version. Please verify it. IF you will see this problem again please open another issue.
